docs/execution-providers/TensorRT-ExecutionProvider.md

```diff
@@ -26,8 +26,8 @@ Pre-built packages and Docker images are available for Jetpack in the [Jetson Zo
 
 |ONNX Runtime|TensorRT|CUDA|
 |---|---|---|
-|master|8.4|11.4|
-|1.12|8.4|11.4|
+|main|8.4|11.4|
```

Comment:
since you're touching this file, can you please help update main to 8.5? (we recently updated)
thanks!

Reply:
Yep, done